# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../libpng/64-config-header.diff
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./pngconf.h.orig	2007-01-05 14:58:52.000000000 +0200
+++ ./pngconf.h	2007-01-11 16:58:35.513932000 +0200
@@ -729,6 +729,9 @@
  * PNG_NO_MMX_CODE disables the use of MMX code without changing the API.
  * When MMX code is off, then optimized C replacement functions are used.
 */
+
+#define PNG_NO_ASSEMBLER_CODE
+
 #if defined(PNG_READ_SUPPORTED) && !defined(PNG_NO_ASSEMBLER_CODE)
 #  ifndef PNG_ASSEMBLER_CODE_SUPPORTED
 #    define PNG_ASSEMBLER_CODE_SUPPORTED
